packages/graphiql/package.json

```diff
@@ -35,6 +35,9 @@
       "git add"
     ]
   },
+  "browserify": {
+    "transform": [["babelify", { "presets": ["@babel/preset-env"] }]]
```

Comment:
we should already have a special downstream babel.config.js with our browserslist for babelify to use, beside this file
we should pass -t babelify on the CLI like we were doing before. not sure where that went, or how we went from min.js -> min.js like that. quite some regressions!

Reply:
oh wait, we never used babelify
https://github.com/graphql/graphiql/blob/ac274d3fb81d4e580920357f0563223c87246d23/resources/build.sh

Reply:
Yeah the babelify package was installed but never used. I tried passing it in the CLI but it wasn't picking up the presets, which was when I saw that FAQ on browserify and applying transforms to dependencies. Following the FAQ solution (adding the config to package.json) is what worked for me. Not sure how else to do it.

Reply:
whatever works best, we just need it to pick up our babel.config.js for graphiql, since its also used for tests and for the development watch. That config is different from what is being passed here.

Reply:
I agree, they should be sourced from the same file. Removed the babel presets specified in the babelify config. Babelify should use the babel.config.js for graphiql now.